spidermon/contrib/validation/cerberus/validator.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import
+from collections.abc import Mapping
+
+from cerberus.validator import (
+    Validator as cerberus_validator,
+    DocumentError,
+    SchemaError,
+)
+from spidermon.contrib.validation.validator import Validator
+from .translator import CerberusMessageTranslator
+
+
+class CerberusValidator(Validator):
+    default_translator = CerberusMessageTranslator()
+    name = "Cerberus"
+
+    def __init__(self, schema, translator=None, use_default_translator=True):
+        super(CerberusValidator, self).__init__(
+            translator=translator, use_default_translator=use_default_translator
+        )
+        if schema is not None:
+            self._schema = schema
+        elif self._schema is None:
+            if isinstance(self._schema, Mapping):
+                self._schema = {}
```

Comment:
If `self._schema is None`, `isinstance(self._schema, Mapping)` will always return `False`. So this code will never be executed.

Reply:
Probably this `if` would be easier to read as
```
if schema:
   ...
else:
   ....
```